packages/frontend2/tsconfig.json

```diff
@@ -41,7 +41,8 @@
     "**/*.cjs",
     "**/*.mjs",
     "**/*.js",
-    ".next/types/**/*.ts"
+    ".next/types/**/*.ts",
+    "src/server/features/costs/utils/add-if-definedts"
```

Comment:
lacks `.` before extension - any particular reason for explicit inclusion here?

Reply:
tbh, no clue what happened there but this must have added itself automatically :O